### ISSUE TITLE
We can remove the html files from src/views

### ISSUE THREAD

**Issue by mnichols08** [chingu-voyages/v43-tier1-team-09#156]
**__Task Description__**
As titled we can remove the html files from within our src folder. They are no longer needed for our app to function.

**__Technical Considerations__**
Story.html does have the logic to print a story and perhaps #42 should be finished first.

**__Additional Considerations__**
None that I can think of.

**Comment by mnichols08** [comment]
Hey @Pushkarmondal I actually just closed this issue with my recent pull request #167, but you are welcome to contribute to this project - soon. You can fork at any time but we cannot accept any code at this time because we a team on a Voyage. It is soon over, but before it is we must not accept any outside help. I hope you understand. But it also ends on the 16th of this month. So after that if you wanted to help expand the project, I don't think it would be a problem at all =)

Check out Chingu if you don't know it already.
